### ISSUE TITLE
Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".

### ISSUE THREAD

**Issue by jimCresswell** [AtomLinter/linter-eslint#177]
Still seeing issues with unsafe-eval

Atom: 1.0.2
linter: 1.2.4
linter-eslint: 3.0.1

Stacktrace:
EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".
  at Function.wrapfunction (/Users/n449892/code/git/specs/node_modules/express/node_modules/depd/index.js:421:5)
  at Object.<anonymous> (/Users/n449892/code/git/specs/node_modules/express/node_modules/send/index.js:156:47)
  at Module._compile (module.js:452:26)
  at Object.loadFile [as .js] (/Users/n449892/Downloads/Atom.app/Contents/Resources/app.asar/src/babel.js:162:21)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/n449892/code/git/specs/node_modules/express/lib/utils.js:19:12)
  at Module._compile (module.js:452:26)
  at Object.loadFile [as .js] (/Users/n449892/Downloads/Atom.app/Contents/Resources/app.asar/src/babel.js:162:21)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/n449892/code/git/specs/node_modules/express/lib/view.js:19:13)
  at Module._compile (module.js:452:26)
  at Object.loadFile [as .js] (/Users/n449892/Downloads/Atom.app/Contents/Resources/app.asar/src/babel.js:162:21)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/n449892/code/git/specs/node_modules/express/lib/application.js:22:12)
  at Module._compile (module.js:452:26)
  at Object.loadFile [as .js] (/Users/n449892/Downloads/Atom.app/Contents/Resources/app.asar/src/babel.js:162:21)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/n449892/code/git/specs/node_modules/express/lib/express.js:17:13)
  at Module._compile (module.js:452:26)
  at Object.loadFile [as .js] (/Users/n449892/Downloads/Atom.app/Contents/Resources/app.asar/src/babel.js:162:21)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/n449892/code/git/specs/node_modules/express/index.js:11:18)
  at Module._compile (module.js:452:26)
  at Object.loadFile [as .js] (/Users/n449892/Downloads/Atom.app/Contents/Resources/app.asar/src/babel.js:162:21)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/n449892/code/git/specs/app.js:3:15)
  at Module._compile (module.js:452:26)
  at Object.loadFile [as .js] (/Users/n449892/Downloads/Atom.app/Contents/Resources/app.asar/src/babel.js:162:21)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at /Users/n449892/code/git/specs/node_modules/eslint/lib/load-rules.js:36:36
  at Array.forEach (native)
  at module.exports (/Users/n449892/code/git/specs/node_modules/eslint/lib/load-rules.js:32:30)
  at Object.load (/Users/n449892/code/git/specs/node_modules/eslint/lib/rules.js:42:20)
  at /Users/n449892/code/git/specs/node_modules/eslint/lib/cli-engine.js:271:19
  at Array.forEach (native)
  at new CLIEngine (/Users/n449892/code/git/specs/node_modules/eslint/lib/cli-engine.js:269:32)
  at Object.provider.lint (/Users/n449892/.atom/packages/linter-eslint/lib/linter-eslint.coffee:85:24)
  at /Users/n449892/.atom/packages/linter/lib/editor-linter.coffee:69:26
  at currentLinter (/Users/n449892/.atom/packages/linter/lib/editor-linter.coffee:68:20)
  at /Users/n449892/.atom/packages/linter/lib/editor-linter.coffee:82:23
  at Set.forEach (native)
  at EditorLinter.triggerLinters (/Users/n449892/.atom/packages/linter/lib/editor-linter.coffee:64:26)
  at /Users/n449892/.atom/packages/linter/lib/editor-linter.coffee:57:27

Config
```JSON
  "linter-eslint":
    showRuleIdInMessage: true
    eslintRulesDir: "./"
  linter: {}
```

Occurred on a file with the following content

```javascript
/**
 * Entry point for defining Gulp tasks.
 */

// Pull in the tasks from the gulp-tasks directory.
require('require-dir')('./gulp-tasks');

```

**Comment by ide** [comment]
I'm seeing this too. linter-eslint correctly finds my eslint package under `node_modules`. This error occurs when it tries to require eslint... this looks like the culprit https://github.com/eslint/eslint/issues/3163.

**Comment by jimCresswell** [comment]
Great!

**Comment by ide** [comment]
Installing this package from master should now pull in the latest copy of loophole that fixes this issue. 

@iam4x, could you publish a new version to npm after taking a look at my PR to update the package.json dependency? I believe that will fix this issue for everyone using eslint 1.0rc2.
